lib/services/plugins-service.ts

```diff
@@ -182,11 +169,11 @@ export class PluginsService implements IPluginsService {
 		return installedPlugin !== null && installedPlugin !== undefined;
 	}
 
-	public configurePlugin(pluginName: string, version: string): IFuture<void> {
+	public configurePlugin(pluginName: string, version: string, configurations?: string[]): IFuture<void> {
 		return (() => {
-			if(this.$project.hasBuildConfigurations()) {
-				var configurations = this.$project.configurations;
-				_.each(configurations, (configuration:string) => {
+			var configs = configurations || this.$project.configurations;
+			if(configs) {
```

Comment:
ahh, configs cannot be falsy, but my implementation is incorrect, I should check for ```this.$project.hasBuildConfigurations()```
I'll fix this